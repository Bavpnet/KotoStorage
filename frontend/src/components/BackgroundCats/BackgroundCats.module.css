.bgCats {
  background-image: url('./../../assets/bg_cats.png');
  height: 1024px;
  width: 100%;
}

.wrapperOpacity {
  background-color: rgba(228, 214, 192, 0.65);
  height: 100%;
  width: 100%;
  padding-top: 64px;
  display: flex;
  justify-content: center;
}
