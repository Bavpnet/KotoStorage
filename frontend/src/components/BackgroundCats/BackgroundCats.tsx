import { PropsWithChildren } from 'react'
import styles from './BackgroundCats.module.css'

export const BackgroundCats: React.FC<PropsWithChildren> = ({ children }) => {
  return (
    <div className={styles.bgCats}>
      <div className={styles.wrapperOpacity}>{children}</div>
    </div>
  )
}
