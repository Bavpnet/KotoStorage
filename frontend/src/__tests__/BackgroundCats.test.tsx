import renderer from 'react-test-renderer'
import { BackgroundCats } from '../components/BackgroundCats'

describe('BackgroundCats', () => {
  it('renders correctly', () => {
    const tree = renderer.create(<BackgroundCats />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
