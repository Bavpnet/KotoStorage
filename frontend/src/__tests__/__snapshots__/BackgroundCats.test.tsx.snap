// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackgroundCats renders correctly 1`] = `
<div
  className="bgCats"
>
  <div
    className="wrapperOpacity"
  />
</div>
`;
